README.md

```diff
@@ -1,31 +1,55 @@
-# angular-a-disabled
```

Comment:
may-be we can add :skull: or :skull_and_crossbones: or :warning: to say it's deprecated